packages/duf/src/evo/data_converters/duf/importer/duf_attributes_to_evo.py

```diff
@@ -0,0 +1,135 @@
+            attributes_go.append(attribute_go)
+
+    return attributes_go
+
+
+def convert_duf_single_value_attribute(
+    attribute: tuple[str, Any], data_client: ObjectDataClient
+) -> Optional[OneOfAttribute_V1_2_0_Item]:
+    key, value = attribute
+
+    if isinstance(value, str):
+        try:
+            value = isoparse(value)
+        except ValueError:
+            pass
+
+    match value:
+        case str():
+            table = pa.table(
+                [[value]],
+                schema=pa.schema(
+                    [
+                        pa.field("n0", pa.string()),
+                    ]
+                ),
+            )
+            table = data_client.save_table(table)
+            return StringAttribute_V1_1_0(
+                name=key,
+                key=key,
+                values=StringArray_V1_0_1(
+                    data=table["data"],
+                    length=1,
+                ),
+            )
+        case int():
+            table = pa.table(
+                [[value]],
+                schema=pa.schema(
+                    [
+                        pa.field("n0", pa.int64()),
+                    ]
+                ),
+            )
+            table = data_client.save_table(table)
+            return IntegerAttribute_V1_1_0(
+                name=key,
+                key=key,
+                values=IntegerArray1_V1_0_1(
+                    data=table["data"],
+                    length=1,
+                    data_type=table["data_type"],
+                ),
+                nan_description=NanCategorical_V1_0_1(values=[]),
+            )
+        case float():
+            table = pa.table(
+                [[value]],
+                schema=pa.schema(
+                    [
+                        pa.field("n0", pa.float64()),
+                    ]
+                ),
+            )
+            table = data_client.save_table(table)
+            return ContinuousAttribute_V1_1_0(
+                name=key,
+                key=key,
+                values=FloatArray1_V1_0_1(
+                    data=table["data"],
+                    length=1,
+                    data_type=table["data_type"],
+                ),
+                nan_description=NanContinuous_V1_0_1(values=[]),
+            )
+        case datetime():
+            table = pa.table(
+                [[value]],
+                schema=pa.schema(
+                    [
+                        pa.field("n0", pa.timestamp("us", tz="UTC")),
+                    ]
+                ),
+            )
+            table = data_client.save_table(table)
+            return DateTimeAttribute_V1_1_0(
+                name=key,
+                key=key,
+                values=DateTimeArray_V1_0_1(
+                    data=table["data"],
+                    length=1,
+                    data_type=table["data_type"],
+                ),
+                nan_description=NanCategorical_V1_0_1(values=[]),
+            )
```

Comment:
This is a good place to use a factory pattern for converting by type. Not a request for change, just a general comment.